test/sequential/test-fs-watch.js

```diff
@@ -78,7 +73,9 @@ setImmediate(function() {
   fs.writeFileSync(filepathTwoAbs, 'pardner');
 });
 
-fs.mkdirSync(testsubdir, 0o700);
+var filenameThree = 'newfile.txt';
```

Comment:
Sure, I was just trying to keep the modifications to a minimum. Should I use `const` everywhere on the test or just in the variables I moved?